src/webview/KaotoEditorEnvelopeApp.ts

```diff
@@ -1,10 +1,10 @@
-import { KaotoEditorFactory } from "@kaoto/kaoto-ui/dist/lib/kogito-integration";
-import * as EditorEnvelope from "@kie-tools-core/editor/dist/envelope";
-
+import { KaotoEditorFactory } from '@kaoto/kaoto-ui/dist/lib/kogito-integration';
+import { Editor, EditorFactory, KogitoEditorChannelApi } from '@kie-tools-core/editor/dist/api';
+import * as EditorEnvelope from '@kie-tools-core/editor/dist/envelope';
 declare const acquireVsCodeApi: any;
 
 EditorEnvelope.init({
-  container: document.getElementById("envelope-app")!,
+  container: document.getElementById('envelope-app')!,
   bus: acquireVsCodeApi(),
-  editorFactory: new KaotoEditorFactory(),
+  editorFactory: new KaotoEditorFactory() as unknown as EditorFactory<Editor, KogitoEditorChannelApi>,
```

Comment:
There's a mismatch between `@kie-tools-core` from `kaoto-ui` and `vscode-kaoto`. This will be removed during the next release of `kaoto-ui`.

Reply:
So related to the comment here https://github.com/KaotoIO/vscode-kaoto/pull/273/files?diff=split&w=1#r1229554845

if I understand well, this is a trick to avoid failure at compile time but in case a method that does not exists is called it will fail at runtime?